numpyro/distributions/transforms.py

```diff
@@ -611,7 +611,7 @@ class LowerCholeskyAffine(Transform):
        >>> scale_tril = jnp.array([[0.3, 0.0], [1.0, 0.5]])
        >>> affine = LowerCholeskyAffine(loc=loc, scale_tril=scale_tril)
        >>> affine(base)
-       DeviceArray([0.3, 1.5], dtype=float32)
+       Array([0.3, 1.5], dtype=float32)
```

Comment:
Sorry, @YanniPapandreou! This change is needed for jax 0.4 but currently, CI is running with python 3.7 which is not compatible with jax 0.4. So CI is installing jax < 0.4. Could you revert this change?

Reply:
@fehiepsi done and pushed!